ts/testUtils.ts

```diff
@@ -164,6 +164,37 @@ export const buildPackage = (
   return Deno.readLinkSync(`${dir}/result`);
 };
 
+export const runInDevShell = (
+  devShell: garn.Environment,
+  options: { cmd: string; dir?: string } = { cmd: "true" },
+): void => {
+  const dir = options.dir ?? Deno.makeTempDirSync({ prefix: "garn-test" });
```

Comment:
Nit, maybe it's nice to default this the same way we do dir? It's kind of sad that typescript doesn't have great syntax for defaulting object properties

```suggestion
export const runInDevShell = (
  devShell: garn.Environment,
  options: { cmd?: string; dir?: string } = {},
): void => {
  const cmd = options.cmd ?? "true";
  const dir = options.dir ?? Deno.makeTempDirSync({ prefix: "garn-test" });
```